jekyll/_cci2/proxy.adoc

```diff
@@ -111,22 +100,19 @@ CircleCI uses `curl`  and `awscli` scripts to download initialization scripts, a
 * If you are using Docker refer to the https://docs.docker.com/engine/admin/systemd/#/http-proxy[Docker HTTP Proxy Instructions] documentation.
 * If you are running a static installation, add the following to the server before installation:
 +
-```bash
+
+```
 #!/bin/bash
 
-(cat <<'EOF'
+(cat <<EOF
 HTTP_PROXY=<proxy-ip:port>
 HTTPS_PROXY=<proxy-ip:port>
-NO_PROXY=169.254.169.254,<circleci-service-ip>,
-127.0.0.1,localhost,ghe.example.com
-JVM_OPTS="-Dhttp.proxyHost=<ip> -Dhttp.proxyPort=<port>
--Dhttps.proxyHost=<proxy-ip> -Dhttps.proxyPort=3128 -Dhttp.nonProxyHosts=169.254.169.254|<circleci-service-ip>|
-127.0.0.1|localhost|ghe.example.com"
-EOF
-) | sudo tee -a /etc/environment
+NO_PROXY=169.254.169.254,<circleci-service-ip>,127.0.0.1,localhost,ghe.example.com
+JVM_OPTS="-Dhttp.proxyHost=<ip> -Dhttp.proxyPort=<port> -Dhttps.proxyHost=<proxy-ip> -Dhttps.proxyPort=3128 -Dhttp.nonProxyHosts=169.254.169.254|<circleci-service-ip>|127.0.0.1|localhost|ghe.example.com"
+EOF) | sudo tee -a /etc/environment
 
 set -a
-. /etc/environment
+./etc/environment
```

Comment:
I believe we still need a space ` ` between `.` and `/`, because this code is to load `/etc/environment`.

Reply:
Ah, Thank you! I misunderstand this. and add a fix for this